src/styles/blocks/style-block-stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: grid;
+  grid-auto-flow: column;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    border-radius: 0.5px;
+    background-position: center;
+  }
+
+  &--4 :nth-child(-n+4) {
```

Comment:
The modifier class should be applied to the star elements, not the container. Change &--4 to &__star--4 and move it inside &__star block.